.github/ISSUE_TEMPLATE/good-first-issue.md

```diff
@@ -1,14 +0,0 @@
----
-name: Good First Issue
```

Comment:
I don't know if this has ever been used. It's also more something for the core maintainers to decide.